code/modules/client/preferences_savefile.dm

```diff
@@ -5,7 +5,7 @@
 //	You do not need to raise this if you are adding new values that have sane defaults.
 //	Only raise this value when changing the meaning/format/name/layout of an existing value
 //	where you would want the updater procs below to run
-#define SAVEFILE_VERSION_MAX	38
+#define SAVEFILE_VERSION_MAX	39
```

Comment:
see above comment. you don't need a version bump since you have a sane default

Reply:
changed back to 38